workspaces/src/network/mod.rs

```diff
@@ -120,12 +120,12 @@ where
     async fn patch_state(
         &self,
         contract_id: &AccountId,
-        key: String,
+        key: Vec<u8>,
         value: Vec<u8>,
```

Comment:
these should probably be slices to be consistent with the rest of the API, or is there a reason they are not @ChaoticTempest ?

Reply:
ahh, yeah best to keep them as slices. My bad, prematurely merged this one since it looked simple

Reply:
no worries, I can fix this real quick, I just went by the issue description

you want me to change both key and value to be [u8], right?

Reply:
yup both should be slices

Reply:
was unrelated to this PR, just wanted to point out before opening an issue :)